x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial.csv-spec

```diff
@@ -1203,7 +1203,7 @@ count:long | country:k
 1          | Poland
 ;
 
-airportsSortCityName
+airportsSortCityName#[skip:-8.13.3, reason:fixed in 8.13]
```

Comment:
I thought we were able to use capabilities even for older versions, so we could get rid of all cases of this hash-skip?

Reply:
If there's an appropriate capability, that would've worked, too! The correct one is harder to find than the correct version from which we can successfully run the test, and for the bwc tests the effect is the same.